### ISSUE TITLE
Deployment failure

### ISSUE THREAD

**Issue by ADD-SP** [jenkey2011/vuepress-deploy#22]
This is the configuration.
```
- name: Build & Deploy
  uses: jenkey2011/vuepress-deploy@master
  env:
    ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
    TARGET_BRANCH: gh-pages
    BUILD_SCRIPT: set -e && yarn && yarn docs:build
    BUILD_DIR: docs/.vuepress/dist/
```

Below is the error log for Github Action.
```
$ npx vuepress build docs
wait Extracting site metadata...
Error: EACCES: permission denied, mkdir '/github/workspace/node_modules/@vuepress/core/.temp'
```

It doesn't seem like a big deal.

**Comment by jenkey2011** [comment]
It seems OK now? 
https://github.com/ADD-SP/ngx_waf/runs/3068265479?check_suite_focus=true

**Comment by ADD-SP** [comment]
> 
> 
> It seems OK now?
> https://github.com/ADD-SP/ngx_waf/runs/3068265479?check_suite_focus=true

No, the link in your reply uses `jenkey2011/vuepress-deploy@v1.8.1` because the branch `master` was not working so I used the old version.

I just re-ran https://github.com/ADD-SP/ngx_waf/runs/3072917961?check_suite_focus=true and this Action uses `jenkey2011/vuepress-deploy@master`. But the problem is not solved.  The good news is that you can view the full error log at this link.

**Comment by Neunerlei** [comment]
Hi there, 

I can confirm, this is still an issue as you can see here: https://github.com/labor-digital/bits-docs/runs/3342266685?check_suite_focus=true

However, the rollback to v1.8.1 fixed the issue for me, as well :)
